### ISSUE TITLE
libqtile.lazy.LazyCall no such key object Startup Error

### ISSUE THREAD

**Issue by sirakai-pc** [qtile/qtile#1803]
Here's my config error:
`2020-07-04 21:34:34,908 ERROR libqtile qtile.py:make_qtile():L129 Error while reading config file (No such key: <libqtile.lazy.LazyCall object at 0x7f108124cac0>) Traceback (most recent call last): File "/home/sirakai/.local/lib/python3.8/site-packages/libqtile/scripts/qtile.py", line 127, in make_qtile config = confreader.Config.from_file(options.configfile, kore=kore) File "/home/sirakai/.local/lib/python3.8/site-packages/libqtile/confreader.py", line 85, in from_file cnf.load() File "/home/sirakai/.local/lib/python3.8/site-packages/libqtile/confreader.py", line 109, in load self.validate() File "/home/sirakai/.local/lib/python3.8/site-packages/libqtile/confreader.py", line 122, in validate raise ConfigError("No such key: %s" % k.key) libqtile.confreader.ConfigError: No such key: <libqtile.lazy.LazyCall object at 0x7f108124cac0>`

I sincerely don't know what could be the problem, does someone know?..

# Qtile version
0.15.1.dev158+gfb45432b

# Stack traces

# Configuration
[My config.py](https://pastebin.com/JbMwsCWM)

**Comment by ramnes** [comment]
Hello @sirakai-pc!

You're not using `Key` properly. Its signature is `Key(modifiers, key, *commands)`, but you are doing `Key([mod, "Return"], lazy.spawn(myTerm))`. Your code is interpreted as `Key(modifiers=[mod, "Return"], key=lazy.spawn(myTerm))`, thus the error that you see, which is pretty self-explanatory.

TL;DR: you need to do something like this for most of your keys:

```diff
-Key([mod, "Return"], lazy.spawn(myTerm)),  # Open terminal
+Key([mod], "Return", lazy.spawn(myTerm)),  # Open terminal
```

I'll go ahead and close this. Feel free to comment if needed!

**Comment by sirakai-pc** [comment]
Also for keys that aren't "mod"?
For example:
`Key(["control"], "Escape", lazy.spawn("htop")), #Open glances/htop
`

**Comment by sirakai-pc** [comment]
???

**Comment by ramnes** [comment]
No, these are correct.

**Comment by sirakai-pc** [comment]
Thanks.
